osu.Game.Rulesets.Mania/Mods/ManiaModScrollUp.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Mania.Objects;
+using osu.Game.Rulesets.Mania.UI;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.UI;
+using osu.Game.Scoring;
+using osu.Game.Screens.Play;
+using static osu.Game.Rulesets.Mania.ManiaSettingsSubsection;
+
+namespace osu.Game.Rulesets.Mania.Mods
+{
+    public class ManiaModScrollUp : Mod, IApplicableToDrawableRuleset<ManiaHitObject>, IApplicableToScoreProcessor, IApplicableToPlayer
+    {
+        public override string Name => "Scroll up";
+        public override string Acronym => "SU";
+        public override LocalisableString Description => @"Key will become faster..., until you miss";
+        public override double ScoreMultiplier => 1;
+        public override IconUsage? Icon => FontAwesome.Solid.AngleDoubleUp;
```

Comment:
remove icon for now. can't find it now but it was decided in one of these that new mods should not be choosing their own icons